src/redux/user/user.sagas.js

```diff
@@ -75,6 +76,38 @@ export const loginUser = (data) => {
   return setItems(query, data);
 };
 
+export function* handleGoogleUserLogin({ payload }) {
+  try {
+    yield put(setUserLoading(true));
+    const user = yield call(
+      getItems,
+      `
+    mutation($id_token:String!){googleUser(id_token:$id_token){
+   firstName,
+      lastName,
+      email,
+      credentials{
+        source,
+        tokenPass
+      }
+  token
+} 
+
+}
```

Comment:
Align this mutation to make it more readable

Reply:
fixed